.github/workflows/upload-nightly-release.yaml

```
@@ -0,0 +1,47 @@
+name: Build nightly PennyLane releases for TestPyPI
+
+on:
+  schedule:
+    # Run every weekday at 23:40 EDT (cron is in UTC)
+    - cron: "40 3 * * 2-6"
+  workflow_dispatch:
+
+jobs:
+  setup:
+    name: Setup the release
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout PennyLane repo
+      uses: actions/checkout@v4
+      with:
+        ssh-key: ${{ secrets.NIGHTLY_VERSION_UPDATE_DEPLOY_KEY }}
+
+    - name: Bump dev version
+      run: |
+        python $GITHUB_WORKSPACE/.github/workflows/scripts/set_nightly_version.py
+
+    - name: Push new version
+      run: |
+        git config --global user.email '${{ secrets.AUTO_UPDATE_VERSION_RINGO_EMAIL }}'
+        git config --global user.name "ringo-but-quantum"
+        git add $GITHUB_WORKSPACE/pennylane/_version.py
+        git commit -m "[no ci] bump nightly version"
```

Does `[no ci]` work? I've always used `[skip ci]`. If it works, today I've learned something new 😄 

```suggestion
        git commit -m "[skip ci] bump nightly version"
```

---

Yes there is actually a few variants you can use :) (I just picked the shortest one 😅)
https://docs.github.com/en/actions/managing-workflow-runs/skipping-workflow-runs

---

That will work. There are a few more options actually. You can find them [here](https://docs.github.com/en/actions/managing-workflow-runs/skipping-workflow-runs) :)